gradle.properties

```diff
@@ -21,3 +21,5 @@ libVcsUrl=https://github.com/mozilla/application-services.git
 
 libLicense=MPL-2.0
 libLicenseUrl=https://www.mozilla.org/en-US/MPL/2.0/
+
+android.useAndroidX=true
```

Comment:
Hmm... Are we allowed to add this? IIRC androidX causes problems for some of our dependencies?

Reply:
It seems that we use it already for glean in tests (`ApplicationProvider`) :/ I'll make sure it works